src/GitVersionCore.Tests/IntegrationTests/MainlineDevelopmentMode.cs

```diff
@@ -522,6 +522,27 @@ public void VerifyIncrementConfigIsHonoured()
             fixture.AssertFullSemver("3.1.3", minorIncrementConfig);
             Console.WriteLine(fixture.SequenceDiagram.GetDiagram());
         }
+
+        [Test]
+        public void Issue2323()
+        {
+            using var fixture = new EmptyRepositoryFixture();
+            fixture.MakeACommit("Release v1.0.0");
+            fixture.ApplyTag("1.0");
+            fixture.AssertFullSemver("1.0.0", config);
+
+            fixture.BranchTo("release/1.1");
```

Comment:
What happens if you name the release branch `release/1.1.0`?

Reply:
It's the same result, unfortunately